exonum-java-binding-common/src/main/proto/block.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+option java_package = "com.exonum.binding.common.serialization";
```

Comment:
Not sure about package location. `serialization` looks strange for model classes What do you think about `com.exonum.binding.common.model `or `com.exonum.binding.common.schema` ?

Reply:
Seems ok to me. We have the same package location for `TestProtos` and for other protos we don't make a separate package as well